### PR TITLE
Added user story 1.1

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**
User story 1.1 repurposed to be "Register" user story, as child of "Account system" epic. Describes user registration.

### CODE REVIEW

File: magento/user_stories.md [hunk 1]
```diff
@@ -2,7 +2,28 @@
...
+### Scenario 1.1a Successful register
+**Given**: No account has been previously registered using the specified email  
+**When**: The user enters their first name, last name, email address and password into the registration form
+**And**: The user clicks on a "Create account" button
+**Then**: The account is created and the user is logged into the system with that account.
+### Scenario 1.1b Email taken
+**Given**: An account has already been registered using the specified email  
+**When**: The user enters their first name, last name, email address and password into the registration form
+**And**: The user clicks on a "Create account" button
+**Then**: An error message regarding the existing account is displayed.
```
Comment (Stepan0806) [inline magento/user_stories.md:16]: Given-when-then construction is displayed in one line. Adding newline symbols is needed

**Review by Stepan0806 — CHANGES_REQUESTED** [review]

File: magento/user_stories.md (hunk 1 above)
Comment (Marko-Shevchuk) [reply magento/user_stories.md:16]: Double spaces at the end of lines count as newline symbols. Reformat to newlines?

Comment (Stepan0806) [reply magento/user_stories.md:16]: Yes

Comment (Marko-Shevchuk) [reply magento/user_stories.md:16]: All other docs have been reformatted to double-space, do I leave this as-is?

Comment (Stepan0806) [reply magento/user_stories.md]: I've already fixed it

**Review by Stepan0806 — APPROVED** [review]

**Pull request merged**